src/jit/lsra.cpp

```diff
@@ -3949,6 +3962,16 @@ void LinearScan::setIntervalAsSplit(Interval* interval)
 //
 void LinearScan::setIntervalAsSpilled(Interval* interval)
 {
+#if FEATURE_PARTIAL_SIMD_CALLEE_SAVE
+    if (interval->isUpperVector)
```

Comment:
Is it more beneficial to just load/store the upper bits? Looking at recent numbers from Agner's it looks like `VINSERT/VEXTRACT` (from/to memory) have nearly twice the latency as compared to `VMOVU`

Reply:
Also, has there been any thought on "spilling" these to XMM6-XMM15; rather than to the stack (which are considered "callee preserved")?

Reply:
The register allocator will always try to allocate a callee-save register to extract the upper half to, if it can find one. Changing it to always spill the entire vector if it can't is something we could consider, but I would not entertain that as part of this change.

Reply:
For Skylake the tables give quite similar latencies: VMOVDQU=3, VEXTRACT=4, VINSERT=3; and identical throughput, so not too bad.